src/components/image/_image.scss

```diff
@@ -1,9 +1,9 @@
-.ons-figure {
+.ons-image {
   margin: 0;
 
   @extend .ons-u-mb-m;
 
-  &__image {
+  &__img {
```

Comment:
Why does this need to be changed?

Reply:
As the ticket suggested renaming the `ons-figure` to match the component name `ons-image` being the block with the element as `__img` made more sense than repeating `__image`.